assets/src/scss/layout/_breadcrumbs.scss

```diff
@@ -31,7 +31,6 @@
   .tag-wrap {
     display: inline;
     position: relative;
-    color: $link-color;
```

Comment:
I checked which things have the `tag-wrap` class, and all of them only have `tag-item` children, which also set the same color. So we can safely remove this one.